runtime/tests/jniarg/cdefs.c

```diff
@@ -139,7 +139,7 @@ void JNICALL Java_JniArgTests_logRetValError( JNIEnv *p_env, jobject p_this, jst
 	retValFailures ++;
 }
 
-void JNICALL Java_JniArgTests_summary( JNIEnv *p_env, jobject p_this )
+jint JNICALL Java_JniArgTests_summary( JNIEnv *p_env, jobject p_this )
```

Comment:
Why not jboolean?

Reply:
This is the return code the Java program should return. We can also make it a boolean, then we'd have more code to write on the Java side, i.e. check the boolean and return an integer exit code. Do you think this warrants a change?

Reply:
It's test code so I'm mostly indifferent. Let's leave it alone.